lightnode.go

```diff
@@ -141,6 +138,35 @@ func New(options Options, ctx context.Context, logger logrus.FieldLogger, sqlDB
 	)
 
 	watchers := map[multichain.Chain]map[multichain.Asset]watcher.Watcher{}
+
+	// solClient := solanaRPC.NewClient(bindingsOpts.Chains[multichain.Solana].RPC.String())
+	// for _, selector := range options.Whitelist {
```

Comment:
Is there a reason we have disabled watchers?

Reply:
I have removed this commented code, it was from resolving some conflicts. The watcher initialization happens below this commented code.